internal/config/config.go

```diff
@@ -54,9 +59,10 @@ type Config struct {
 	ConfigPath     string
 	ShowFavourites bool
 	Version        string
-	ConfigDir      string `yaml:"-"`
-	Pager          string `yaml:"pager,omitempty"`
-	Feeds          []Feed `yaml:"feeds"`
+	ConfigDir      string  `yaml:"-"`
+	Pager          string  `yaml:"pager,omitempty"`
+	Feeds          []Feed  `yaml:"feeds"`
+	General        General `yaml:"general"`
```

Comment:
Other general config is just at the root level of the config struct, we can keep sort order here too so no need for `General`. 

```
type Config struct {
  DefaultSort string `yaml:"defaultSort"
```